src/ui/public/es.js

```diff
@@ -2,6 +2,12 @@ import 'elasticsearch-browser';
 import _ from 'lodash';
 import uiModules from 'ui/modules';
 
+/**
+ * @name es
+ * @description This is the result of calling esFactory. Where is esFactory
```

Comment:
To keep this PR consistent, can we place a line break between name and description?